sdk/Sdk/Sdk.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <PatchProductVersion>4</PatchProductVersion>
+    <MinorProductVersion>1</MinorProductVersion>
```

Comment:
[Current Worker sdk](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Sdk/) version is 1.0.4 . This change will create 1.1.0.